testapp/src/main/java/com/rakuten/tech/mobile/testapp/ui/display/MiniAppDisplayActivity.kt

```diff
@@ -131,6 +132,7 @@ class MiniAppDisplayActivity : BaseActivity(), PreloadMiniAppWindow.PreloadMiniA
             context.startActivity(Intent(context, MiniAppDisplayActivity::class.java).apply {
                 putExtra(appUrlTag, appUrl)
                 putExtra(updateTypeTag, updatetype)
+                putExtra(isFromMiniAppByUrlActivityTag, true)
```

Comment:
Is it possible to pass this parameter through `startUrl(...)`, so it would be expressible to outside of this class e.g.

```
        fun startUrl(
            context: Context,
            appUrl: String,
            miniAppSdkConfig: MiniAppSdkConfig? = null,
            updatetype: Boolean = false,
            isFromMiniAppByUrlActivity: Boolean = true
        ) { ... }
```

Reply:
I thought about it this way, it just I don't think it could scale to the point that it needs to be this way, ok changed.